admin/resources/js/components/form/Checklist.tsx

```diff
@@ -72,7 +72,8 @@ const Checklist: React.FunctionComponent<ChecklistProps> = ({
             hideOptional
           >
             <input
-              style={{ order: -1 }}
+              style={{ order: -1, marginBottom: "0.5rem" }}
```

Comment:
Why do you need to set the margin-bottom value the style attribute if you are also doing it in the data-h2-margin attribute's value?

Reply:
Hydrogen wouldn't let me set the bottom margin and right margin with 2 different values.. so I threw this in there to line up the checkbox. I guess I can move the bottom from the data-h2-margin attribute.

Reply:
I see. I would recommended using inline then, otherwise it makes it hard to read.